android/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
         <activity
             android:name="com.mygdx.minigolf.AndroidLauncher"
             android:label="@string/app_name" 
-            android:screenOrientation="landscape"
+            android:screenOrientation="portrait"
```

Comment:
Er dette ment å være permanent eller kun for testing?

Reply:
Kun for testing.